### PR TITLE
Stage

### PR DESCRIPTION
**Pull request by raphmur**
Main PR to merge Stage changes (v0.0.11 devs & docs) to main

> [!CAUTION]
> This PR will trigger a new update in Home Assistant

> [!WARNING]
> Merge everything in _core then pull latest changes here.
> check the local files (changelog, readme, docs, config, Dockerfile, libproduct) and merge this PR.

### CODE REVIEW

**Review by baylanger — DISMISSED** [review]

File: tesla_ble_mqtt/app
Comment (raphmur) [inline tesla_ble_mqtt/app:1]: We will need to git pull, at last minute before merging this PR
@baylanger

**Review by raphmur — COMMENTED** [review]
All good to me

File: README.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ Notes to developers after forking or using the github template feature:
 [aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
 [amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
 
-[installations-shield-stable]: https://img.shields.io/badge/dynamic/json?url=https://analytics.home-assistant.io/addons.json&query=$["d5cb030e_tesla_local_commands"].total&label=Reported%20Installations&link=https://analytics.home-assistant.io/add-ons
+[installations-shield-stable]: https://img.shields.io/badge/dynamic/json?url=https://analytics.home-assistant.io/addons.json&query=$["da013fb0_tesla_local_commands"].total&label=Reported%20Installations&link=https://analytics.home-assistant.io/add-ons
```
Comment (raphmur) [inline README.md:42]: to log this, we must keep da013fb0 as the container ID.
So far 10+ installs...

File: tesla_ble_mqtt/libproduct.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  export ENABLE_HA_FEATURES=""
+  export PRESENCE_EXPIRE_TIME_LIST=""
+
+}
+
+#
+# Definition functions to call bashio::log
+#
+function log_debug() {
+  bashio::log.debug "$1"
```
Comment (raphmur) [inline tesla_ble_mqtt/libproduct.sh:55]: I wonder if we would need $@ instead of $1?

**Review by raphmur — COMMENTED** [review]
All good (again)

File: tesla_ble_mqtt/libproduct.sh (hunk 2 above)
Comment (baylanger) [reply tesla_ble_mqtt/libproduct.sh:55]: $1 is enough cuz everything coming here is wrapped with " " or ' ' - it's a message. Personally I've never seen a log not properly wrapped to make it 1 argument.

**Review by baylanger — APPROVED** [review]
🚀🚀🚀  Let the fun start  🚀🚀🚀

**Review by raphmur — COMMENTED** [review]
cannot approve but seems ok

**Pull request merged**